src/scss/views/widgets/_pill.scss

```diff
@@ -0,0 +1,59 @@
+.pill {
+  display: flex;
+  align-items: center;
+  width: fit-content;
+  height: 32px;
```

Comment:
I think we should not hardcode this, as if we choose a larger font-size or wtv, the height should just adjust

Reply:
Ok I think I can achieve the same thing via margin/padding